tests/providers/amazon/aws/hooks/test_emr.py

```diff
@@ -29,8 +30,8 @@
     mock_emr = None
 
 
-@unittest.skipIf(mock_emr is None, 'moto package not present')
```

Comment:
Nice! Love the conversion to pytest here.

Reply:
Yeah, I try to replace `unittests` by `pytest` in most cases this is not required huge afford.
However still a lot of `unittests` tests

```shell
# Total number of files which use `unittest.TestCase `
❯ grep -rl 'unittest.TestCase' ./tests | wc -l
528

# By tests packages
❯ grep -rl 'unittest.TestCase' ./tests | cut -d"/" -f3 | sort | uniq -c | sort -nr
    379 providers
     36 charts
     23 utils
     20 cli
     10 ti_deps
     10 api_connexion
      6 sensors
      6 operators
      6 executors
      6 core
      5 www
      5 always
      4 models
      3 api
      2 task
      2 kubernetes
      1 security
      1 plugins
      1 macros
      1 hooks
      1 dag_processing

# By provider ("apache", "microsoft", "common" has subpackages which is separate provider)
❯ grep -rl 'unittest.TestCase' ./tests/providers/ | cut -d"/" -f5 | sort | uniq -c | sort -nr
    113 google
    101 amazon
     32 apache
     26 microsoft
      6 databricks
      4 redis
      4 mysql
      4 alibaba
      3 trino
      3 tableau
      3 qubole
      3 oracle
      3 jenkins
      3 http
      3 docker
      3 atlassian
      3 arangodb
      3 airbyte
      2 yandex
      2 vertica
      2 telegram
      2 sqlite
      2 snowflake
      2 sftp
      2 segment
      2 salesforce
      2 presto
      2 postgres
      2 opsgenie
      2 neo4j
      2 mongo
      2 jdbc
      2 influxdb
      2 imap
      2 grpc
      2 ftp
      2 exasol
      2 discord
      2 dingding
      2 datadog
      2 common
      2 cncf
      2 asana
      1 ssh
      1 singularity
      1 sendgrid
      1 samba
      1 papermill
      1 openfaas
      1 elasticsearch
      1 cloudant
      1 celery
```